vm/actor/src/builtin/multisig/mod.rs

```diff
@@ -169,6 +167,21 @@ impl Actor {
                 ));
             }
 
+            let result_hash = Self::compute_proposal_hash(rt, tx);
+            if result_hash.is_err() {
+                return Err(ActorError::new(
+                    ExitCode::ErrIllegalState,
+                    "Failed to compute proposal hash".to_string(),
+                ));
+            }
+
+            if !params.proposal_hash.eq(&result_hash.unwrap()) {
```

Comment:
Remove `unwrap` here once you've mapped the above err. Also, think you can just do `params.proposal_hash !== &result_hash` instead of using `eq`